lib/cpuinfo.c

```diff
@@ -57,6 +58,7 @@
 #endif
 #ifdef __linux__
 #include <sched.h> /* sched affinity */
+#include <sys/resource.h>
```

Comment:
Remove this inclusion. common.c has this file

Reply:
Done